Rakefile

```diff
@@ -4,21 +4,36 @@ require 'chronic'
 task default: ["notify:expired"]
 
 namespace :notify do
-  pages_url = ENV.fetch("SITE_PAGE_API_URL")
+  pages_urls = [
```

Comment:
For local testing ISTM you might want to be able to override this - having that in the environment might make that easier?

Reply:
Equally you can just add another couple of lines to this array... I have no strong opinions: decoubling this from the matrix builds of Travis was my main aim and I didn't see how you could do that while keeping the individual envvar settings. Unless the envvar value is a comma-separated single string or something that we split on commas when we run the Rake task.